charts/restate-helm/templates/service.yaml

```diff
@@ -10,7 +10,33 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
-  type: {{ .Values.service.type }}
+  ports:
+    - port: 9070
+      name: admin
+    - port: 8080
+      name: ingress
+    - port: 5122
+      name: node
+  clusterIP: None
```

Comment:
This change can’t be done to an existing service. Let’s keep the existing one as is and make a new one called restate discovery. It only need a 5122 port too

Reply:
Good point. I've updated the service definition.